src/visu/pars/Utils/fetchfunctions.ts

```diff
@@ -160,6 +160,47 @@ export function parseVisuXML(stringXML: string): XMLDocument {
     return returnXML;
 }
 
+export function getLastModified(
```

Comment:
Great idea! But we download still the whole file? I rememered the problem, why I inclued the storage: Wago visus like flexRoom use a lot of subvisualisations inside it. In a main site up to 30x times the same subvisu. This would end in slope of many fetch requests.

Reply:
No HEAD only take the header no body is returned.
You can check it in the network tab of the web visu.

Reply:
Top!